src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/TestFiles/RazorSourceGeneratorComponentTests/ScriptTag/Views_Home_Index.html

```diff
@@ -5,7 +5,7 @@
 alert('hello')</script>
 Component:
 1: <script>alert('hello')</script>
-2: <script>alert(&#x27;hello&#x27;)</script>
+2: <script>alert(\u0027hello\u0027)</script>
```

Comment:
Why did u need to make this change?

Reply:
Seems aspnetcore runtime changed in how it encodes text here - before .net 8, it used HTML entities (`&#x27;`), after .net 8, it uses JavaScript encoding when inside a `<script>` tag (`\u0027`). From quick search, this might be the change: https://github.com/dotnet/aspnetcore/pull/50453/files#diff-3382f23d80c9059e5e8b8a9676070b0517383a7a38f2fb72103a837c38bb95aaR183-R187